### PR TITLE
fix(curriculum): changed test text to use should for Information Security and Quality Assurance

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to PR #36860 

This PR makes sure all challenges' tests in the `Information Security and Quality Assurance` section use "should " or "should not".

### CODE REVIEW

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/logging-a-user-out.english.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ Submit your page when you think you've got it right.
 
 ```yml
 tests:
-  - text: Logout route
-    testString: getUserInput => $.get(getUserInput('url')+ '/_api/server.js') .then(data => { assert.match(data, /req.logout/gi, 'You should be call req.logout() in youre /logout route'); }, xhr => { throw new Error(xhr.statusText); })
-  - text: Logout should redirect to the home page /
+  - text: <code>req.Logout</code> should be called in your <code>\/logout</code> route.
```
Comment (moT01) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/logging-a-user-out.english.md]:
```suggestion
  - text: <code>req.Logout</code> should be called in your <code>/logout</code> route.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/logging-a-user-out.english.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ Submit your page when you think you've got it right.
...
+    testString: getUserInput => $.get(getUserInput('url')+ '/_api/server.js') .then(data => { assert.match(data, /req.logout/gi, 'You should be calling req.logout() in your /logout route'); }, xhr => { throw new Error(xhr.statusText); })
+  - text: Logout should redirect to the home page \/.
```
Comment (moT01) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/logging-a-user-out.english.md]:
```suggestion
  - text: Logout should redirect to the home page.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-value-falls-within-a-specific-range.english.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ Choose the minimum range (3rd parameter) to make the test always pass. It should
 
 ```yml
 tests:
-  - text: All tests should pass
+  - text: All tests should pass.
     testString: getUserInput => $.get(getUserInput('url') + '/_api/get-tests?type=unit&n=9').then(data => {assert.equal(data.state,'passed'); }, xhr => { throw new Error(xhr.responseText); })
-  - text: Use approximately(actual, expected, range) - Chose the correct range
+  - text: You should use approximately(actual, expected, range) - You should Chose the correct range.
```
Comment (moT01) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-value-falls-within-a-specific-range.english.md]:
```suggestion
  - text: You should use approximately(actual, expected, range) - You should choose the correct range.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-value-falls-within-a-specific-range.english.md [hunk 4]
```diff
@@ -24,11 +24,11 @@ Choose the minimum range (3rd parameter) to make the test always pass. It should
...
     testString: getUserInput => $.get(getUserInput('url') + '/_api/get-tests?type=unit&n=9').then(data => {  assert.equal(data.assertions[0].method, 'approximately');  assert.equal(data.assertions[0].args[2], 0.5, 'weirdNumbers(0.5) is in the range (0.5, 1.5]. It\'s within 1 +/- 0.5'); }, xhr => { throw new Error(xhr.responseText); })
-  - text: Use approximately(actual, expected, range) - Chose the correct range
+  - text: You should use approximately(actual, expected, range) - You should chose the correct range.
```
Comment (moT01) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-value-falls-within-a-specific-range.english.md]:
```suggestion
  - text: You should use approximately(actual, expected, range) - You should choose the correct range.
```

**Review by moT01 — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@moT01 I have committed your suggested changes.

**Review by moT01 — APPROVED** [review]
🎉 LGTM 🎉

File: curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/how-to-use-passport-strategies.english.md [hunk 5]
```diff
@@ -26,9 +26,9 @@ Submit your page when you think you've got it right. If you're running into erro
 
 ```yml
 tests:
-  - text: All steps correctly implemented in the server.js
+  - text: All steps should be correctly implemented in the server.js.
     testString:  getUserInput => $.get(getUserInput('url')+ '/_api/server.js') .then(data => { assert.match(data, /showLogin:( |)true/gi, 'You should be passing the variable "showLogin" as true to your render function for the homepage'); assert.match(data, /failureRedirect:( |)('|")\/('|")/gi, 'Your code should include a failureRedirect to the "/" route'); assert.match(data, /login[^]*post[^]*local/gi, 'You should have a route for login which accepts a POST and passport.authenticates local'); }, xhr => { throw new Error(xhr.statusText); })
-  - text: A POST request to /login correctly redirects to /
+  - text: A POST request to /login should correctly redirect to /
```
Comment (Manish-Giri) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/advanced-node-and-express/how-to-use-passport-strategies.english.md]:
```suggestion
  - text: A POST request to /login should correctly redirect to /.
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
Add a missing period for consistency.

**Comment by RandellDawson** [comment]
@Manish-Giri I committed your suggested change.

**Review by Manish-Giri — APPROVED** [review]
LGTM 💯

**Pull request merged**